docs/manual/physicslist.md

```diff
@@ -1,46 +1,153 @@
 # Physics list
 
-:::{todo}
+_remage_ uses a custom modular physics list built on top of Geant4’s standard
+physics constructors. It mostly is based on the physics list implemented in
+_MaGe_. It combines:
```

Comment:
I would not reference MaGe, in the end it's fairly standard. also we might revise it soon